clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2846,6 +2846,14 @@ void CodeGenModule::AddGlobalAnnotations(const ValueDecl *D,
     Annotations.push_back(EmitAnnotateAttr(GV, I, D->getLocation()));
 }
 
+// Add "sycl-unique-id" llvm IR attribute for global variables marked with
+// SYCL device_global attribute, and return a unique string using
+// __builtin_sycl_unique_stable_id.
+static void addSYCLUniqueID(llvm::GlobalVariable *GV, const VarDecl *VD) {
+  auto builtinString = SYCLUniqueStableIdExpr::ComputeName(Context, VD);
```

Comment:
@schittir, you need to add llvm::LLVMContext &Context = getLLVMContext(); that is causing your build failures

Reply:
Thank you so much, Soumi! I'm just surprised why it is happening now but never before!

Reply:
Note that it isn't an `LLVMContext`, it is an `ASTContext`, so you need to get one of those.  Additionally, `getLLVMContext` is a function in the `CodeGenModule` type, and this is a static function (so it isn't part of  `CodeGenModule`, and thus doesn't have access to its fields).

I'm guessing wherever you moved this from to be in its own function already had a context in scope.